app/src/main/java/org/gdg/frisbee/android/api/model/Chapter.java

```diff
@@ -99,37 +99,42 @@ public String toString() {
     }
 
     public String getShortName() {
-        return name.replaceAll("GDG ", "");
+        if (shortName == null) {
+            shortName = name.replaceAll("GDG ", "").trim();
+        }
+        return shortName;
     }
 
     @Override
     public int compareTo(Chapter o) {
-        return name.compareTo(o.getName());
+        return getShortName().compareTo(o.getShortName());
```

Comment:
It may be confusing but I had to do it like this. It is because the getter actually does the shortening and trimming when it is first called.